test/common/upstream/upstream_impl_test.cc

```diff
@@ -2457,6 +2630,7 @@ struct TestFilterProtocolOptionsConfig : public Upstream::ProtocolOptionsConfig
 
 // Cluster extension protocol options fails validation when configured for filter that does not
 // support options.
+// This test case can't be converted for V3 API as it is specific for extension_protocol_options
```

Comment:
Can you convert all the tests to the new typed variant for extension protocol options? Ideally we only have the essential cases still doing boosting.

Reply:
@htuch done. thanks for your review.